src/components/ChannelInner.js

```diff
@@ -693,14 +690,7 @@ class ChannelInner extends PureComponent {
           enabled={!this.props.disableKeyboardCompatibleView}
         >
           <ChannelContext.Provider value={this.getContext()}>
-            <SuggestionsProvider
-              handleKeyboardAvoidingViewEnabled={(trueOrFalse) => {
-                if (this._unmounted) return;
-                this.setState({ kavEnabled: trueOrFalse });
-              }}
```

Comment:
`handleKeyboardAvoidingViewEnabled` isn't a prop in SuggestionsProvider and `kavEnabled` wasn't used anywhere else in this file